packages/core/adder/preconditions.ts

```diff
@@ -94,6 +94,8 @@ export async function validatePreconditions<Args extends OptionDefinition>(
         }
     }
     if (preconditionLog.length > 0) {
+        if (isTesting) return;
```

Comment:
Does not match with the changeset: `fix: preconditions are now ignored during tests`.
This does only skip the logging of the preconditions as far as i can tell.

Reply:
thanks! moved it to the top of the function definition, which should prevent them from running.

Reply:
After further consideration: is this really what we want?

Let's assume #338 would be implemented already. Tauri requires rust and cargo to be installed on the dev's machine. Not running the preconditions will not make the dev aware that this is required.

But I can guess where you are coming from. Probably the `clean working directory (Found modified files)` is bothering you. Wouldn't it be enough if we disable that specific precondition during testing?

Reply:
The problem isn't so much the logging, that's fine, it's just that they're causing the tests to fail when testing individual adders with `pnpm test drizzle`